csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -319,19 +319,19 @@ public LogLevel LogVerbosityLevel
         /// Threadpool size for the session.Run() calls. 
         /// Default = 0, meaning threadpool size is aumatically selected from number of available cores.
         /// </summary>
```

Comment:
automatically
If the logic is to use the number of physical cores, we should be a bit more explicit as 'available cores' could mean physical or logic.
Should there be some documentation saying how/when the 'intra op' threads are used? Not sure 'Threadpool size for the Run calls' is enough as both the intra and inter op thread pools could be used in Run.

Reply:
It's hard to get number of physical cores on Linux without using a third-party lib like hwloc